fastfold/utils/tensor_utils.py

```diff
@@ -52,7 +53,8 @@ def dict_multimap(fn, dicts):
         if type(v) is dict:
             new_dict[k] = dict_multimap(fn, all_v)
         else:
-            new_dict[k] = fn(all_v)
+            # when bs = 1, returns [...] rather than [1, ...]
+            new_dict[k] = fn(all_v) if len(all_v) > 1 else all_v[0]
```

Comment:
not sure `all_v[0]` or `fn(all_v[0])`

Reply:
The same.
`all_v` is a list including tensors, and `fn` is `torch.stack` transferring the list to a tensor.

Reply:
Hmm, seems `torch.stack` is a function for a tuple of tensors, not a single tensor. So, it is `all_v[0]`.